src/change_stream.ts

```diff
@@ -411,6 +412,15 @@ export interface ChangeStreamCursorOptions extends AbstractCursorOptions {
   startAtOperationTime?: OperationTime;
   resumeAfter?: ResumeToken;
   startAfter?: boolean;
```

Comment:
sure, but is there any harm in updating just the typescript type here to be correct? since neither of the other tickets would directly address this, and it *is* a one line change

Reply:
Oh yea, in favor of correcting `startAfter`. I was only answering the question on why it's unknown

Reply:
Fixed!